src/test/java/org/mockitousage/basicapi/ResetTest.java

```diff
@@ -26,7 +26,6 @@
 
     @Test
     public void shouldResetOngoingStubbingSoThatMoreMeaningfulExceptionsAreRaised() {
-        mock(IMethods.class);
```

Comment:
I dropped this statement instead of adding `@SuppressWarnings({"CheckReturnValue", "MockitoUsage"})`, but maybe I just don't understand how this statement is relevant to the test. (I would have expected an interaction with the `@Mock`ed field.) Let me know if I should revert this.

Reply:
Yeah this seems fine to me :+1: